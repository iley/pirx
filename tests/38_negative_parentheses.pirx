func main(): int {
  var x: int;
  var y: int;
  x = -(5 + 3);
  y = (-2) * (-4);
  printf("x = %d\n", x);
  printf("y = %d\n", y);
  printf("-(x + y) = %d\n", -(x + y));
  printf("(-x) + (-y) = %d\n", (-x) + (-y));
  printf("-(-(x)) = %d\n", -(-(x)));
  return 0;
}