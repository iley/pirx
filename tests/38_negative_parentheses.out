x = -8
y = 8
-(x + y) = 0
(-x) + (-y) = 0
-(-(x)) = -8
